plugin-template/version.sh

```diff
@@ -0,0 +1,5 @@
+if [ $# -eq 0 ]; then
+	echo "To bump version, pass major/minor/patch "
+else
+	bump2version $1
```

Comment:
can make this $* ? It would allow --current-version

Reply:
also were you going to include git tag stuff here?

Reply:
why do we need `--current-version`? look at `setup.cfg`

Reply:
there are three places you have to edit the current version if you want to change it via editing files.

Reply:
bump2version does git tagging. version.sh is indeed trivial, but the benefit is self-documenting. just looking at a plugin, I wouldn't necessarily know to use bump2version to increment the version and git tag.

it's only two files, setup.cfg for bump2version to keep track, and \_\_init\_\_.py

Reply:
it gives me exceptions if I leave setup.py unchanged

Reply:
this PR removes setup.py...

Reply:
right, my mistake. still, why edit two files when you can just use --current-version? I don't understand the resistance.

Reply:
what do you mean edit two files individually? all you do is `./version.sh patch`. I'm confused

Reply:
I'm thinking of the case where you forgot a git stash and want to create a new local commit, updating the tag to the version you current have assigned to one commit before.

Reply:
I'm not sure I understand. if it's just moving the tag, can't you `git tag -fa v1.0.0` (replacing "v1.0.0" with the latest version tag)?

Reply:
ok I just tested bump2version with current-version and it doesn't even let me go back a version using --current-version so I void my argument here because it doesn't work how I want it to anyway.

Reply:
but yes, you can do that. I just wanted a simple way of reversioning but it looks like that's not a thing with bump2version. I may write something later because when the current version is not right, changing it is a pain and that is what I want this script in particular to handle. I'll approve this PR though, that can be a problem for another time.